CONTRIBUTING.md

```diff
@@ -1,310 +1,31 @@
-./risedev d                        # shortcut for ./risedev dev
-psql -h localhost -p 4566
-```
-
-The default dev cluster includes meta-node, compute-node and frontend-node processes and an embedded volatile in-memory state storage. No data will be persisted. This should be very useful when developing and debugging.
-
-To stop the cluster,
-
-```shell
-./risedev k # shortcut for ./risedev kill
-```
-
-To view the logs,
-
-```shell
-./risedev l # shortcut for ./risedev logs
-```
-
-To clean local data and logs,
-
-```shell
-./risedev clean-data
-```
-
-### Additional Components
-
-RiseDev supports automatic config of some components. We can add components like etcd, MinIO, Grafana, Prometheus and jaeger to the cluster, so as to persist data and monitor the cluster.
-
-To add those components into the cluster, you will need to configure RiseDev to download them at first. For example,
-
-```shell
-./risedev configure                               # start the interactive guide to enable components
-./risedev configure enable prometheus-and-grafana # enable Prometheus and Grafana
-./risedev configure enable minio                  # enable MinIO
-```
-
-After that, you can modify `risedev.yml` to compose the cluster. For example, we can modify the default section to:
-
-```yaml
-  default:
-    - use: minio
-    - use: meta-node
-      enable-dashboard-v2: false
-    - use: compute-node
-    - use: frontend
-    - use: prometheus
-    - use: grafana
-```
-
-Now we can run `./risedev d`. The new dev cluster will contain components from RisingWave, as well as MinIO to persist storage data, and Grafana to monitor the cluster. RiseDev will automatically configure the components to use the available storage service and monitor target.
-
-You may also add multiple compute nodes in the cluster. The `ci-3cn-1fe` config is an example.
-
-### Start Playground with RiseDev
-
-Sometimes, developers might not need to start a full cluster to develop. `./risedev p` can help start an all-in-one process, where meta-node, compute-node and frontend-node are running in the same process. Logs are also printed to stdout instead of separate log files.
-
-```shell
-./risedev p # shortcut for ./risedev playground
-```
-
-For more information, refer to `README.md` under `src/risedevtool`.
-
-### Start Playground with cargo
-
-To start an all-in-one process from IDE or command line, you may also use
-
-```shell
-cargo run --bin risingwave -- playground
-```
-
-Then, connect to the playground instance via
-
-```shell
-psql -h localhost -p 4566
-```
-
-## Testing and Lint
-
-The RisingWave project enforces several checks in CI. Everytime the code gets modified, several checks should be done.
-
-### Lint
+This document describes how to submit your code changes. To learn about the development process, see the [Developer guide](docs/developer-guide.md). To understand the design and implementation of RisingWave, refer to the design docs listed in [readme.md](docs/README.md).
```

Comment:
> Developer guide -> Developer Guide?
> 
> readme.md -> Design Docs Readme

I would change all to be in lower case to comply with our style guide (use upper case only when you have to). It would be 'developer guide' and 'design docs readme.md'. :)